cunumeric/linalg/cholesky.py

```diff
@@ -190,7 +171,7 @@ def choose_color_shape(runtime: Runtime, shape: Shape) -> Shape:
 
 
 def tril_single(context: Context, output: Store) -> None:
-    task = cast(ManualTask, context.create_task(CuNumericOpCode.TRILU))
+    task = context.create_task(CuNumericOpCode.TRILU)
```

Comment:
Could use more explicit `create_auto_task` here if that is preferable.

Reply:
https://github.com/nv-legate/cunumeric/pull/413/commits/875603c4977613cdf78b2ad60eaf4710d6c9d3bd